packages/web3-bzz/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "web3-bzz",
-    "version": "1.9.0",
+    "version": "1.10.0",
```

Comment:
pls revert following changes in this PR:

- package version bumps
- package locks
- all changes in dist dir
- lerna.json changes